client/components/ProductInfo.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class ProductInfo extends React.Component {
+  render() {
+    const { props } = this;
+    return (
+      <div>
+        <p>Purchased item:</p>
```

Comment:
Should 'item' be capitalized? Nope!  (Answered my own question) but leaving this here for fun.  :D